components/templates/m-chart-settings.php

```diff
@@ -40,67 +60,79 @@
-						</p>
-					</td>
-				</tr>
-				<tr>
-					<th scope="row"><?php esc_html_e( 'Numeric Symbol Magnitude', 'm-chart' ); ?></th>
-					<td>
-						<input type="number" name="<?php echo $this->get_field_name( 'numericSymbolMagnitude', 'lang_settings' ); ?>" value="<?php echo absint( $settings['lang_settings']['numericSymbolMagnitude'] ); ?>" />
-						<p class="description">
-							<?php esc_html_e( 'Allows adjustment for languages that use symbols at different intervals (Japanese, Korean, etc...)', 'm-chart' ); ?>
-						</p>
-					</td>
-				</tr>
+								foreach ( m_chart()->library( 'highcharts' )->get_themes() as $theme ) {
+									?>
+									<option value="<?php echo esc_attr( $theme->slug ); ?>"<?php selected( $theme->slug, $settings['default_theme'] ); ?>>
+										<?php esc_html_e( $theme->name, 'm-chart' ); ?>
+									</option>
+									<?php
+								}
+								?>
+							</select>
+							<p class="description">
+								<?php esc_html_e( 'See the M Chart documentation for more info on how to use themes:', 'm-chart' ); ?>
+								<a href="https://github.com/methnen/m-chart/wiki/Themes">https://github.com/methnen/m-chart/wiki/Themes</a>
+							</p>
+						</td>
+					</tr>
+					<?php
+				}
+				?>
 			</tbody>
 		</table>
+		<?php
+		if ( 'highcharts' == $settings['library'] ) {
+			?>
+			<h2><?php esc_html_e( 'Language Settings', 'm-chart' ); ?></h2>
+			<table class="form-table">
+				<tbody>
+					<tr>
+						<th scope="row"><?php esc_html_e( 'Decimal Indicator', 'm-chart' ); ?></th>
+						<td>
+							<input type="text" name="<?php echo $this->get_field_name( 'decimalPoint', 'lang_settings' ); ?>" value="<?php echo esc_attr( $settings['lang_settings']['decimalPoint'] ); ?>" maxlength="1" size="1" />
+						</td>
+					</tr>
+					<tr>
+						<th scope="row"><?php esc_html_e( 'Thousands Separator', 'm-chart' ); ?></th>
+						<td>
+							<input type="text" name="<?php echo $this->get_field_name( 'thousandsSep', 'lang_settings' ); ?>" value="<?php echo esc_attr( $settings['lang_settings']['thousandsSep'] ); ?>" maxlength="1" size="1" />
+						</td>
+					</tr>
+					<tr>
+						<th scope="row"><?php esc_html_e( 'Numeric Symbols', 'm-chart' ); ?></th>
+						<td>
+							<input type="text" name="<?php echo $this->get_field_name( 'numericSymbols', 'lang_settings' ); ?>" value="<?php echo esc_attr( implode( ', ', $settings['lang_settings']['numericSymbols'] ) ); ?>" />
+							<p class="description">
+								<?php esc_html_e( 'Seperate by commas (Thousands, Millions, Billions, Trillions, Quadrillions, Quintillions...)', 'm-chart' ); ?>
+							</p>
+						</td>
+					</tr>
+					<tr>
+						<th scope="row"><?php esc_html_e( 'Numeric Symbol Magnitude', 'm-chart' ); ?></th>
+						<td>
+							<input type="number" name="<?php echo $this->get_field_name( 'numericSymbolMagnitude', 'lang_settings' ); ?>" value="<?php echo absint( $settings['lang_settings']['numericSymbolMagnitude'] ); ?>" />
```

Comment:
Escape name value? (sorry I keep posting the same thing - it'll probably be easier to make sure you catch all of them, though.)